test/main.test.js

```diff
@@ -492,7 +492,7 @@ describe('bundling single js feed', () => {
         supertest(server)
             .post('/bundle/js')
             .send(['completelyfake.json'])
-            .expect(409));
+            .expect(404));
```

Comment:
My feeling on revisiting this is that 404 is more appropriate. It doesn't seem like a conflict. Thoughts?

Reply:
IMO it should be a separate commit.

404 is fine, or just 400

Reply:
422 or 424 maybe? https://tools.ietf.org/html/rfc4918#page-78
404 works as well....

Reply:
Why is 404 more appropriate? Its a bit hard to see in the PR.

Reply:
My complaint with 409 was that we only throw this error when 1 of the feed files to be bundled was not found. 424 seems pretty good actually since these feed files are dependencies of the bundling process.

Reply:
422 and 424 are WebDav status codes. We should restrain from using those. 

To me this seems like a 500. We are able to understand the POST message but in the process of fetching files from underlaying infrastructure, something goes bonkers. Iow; an internal server error.

Reply:
hmm. Only thing that differentiates this particular error is that the hash requested does not exist so the client has actually screwed the pooch a little by asking for the wrong thing. Its a fine line for sure but isnt that a client error rather than a server error?

Reply:
A syntactically valid request is not necessarily a valid request.

in the "you requested a bundle with a feed which does not exist" case, a 5xx is not correct, it's a 4xx error

Reply:
If the file does not exist its a 404. If the file exist, but we get an error reading it, its a 500 in my opinion.

Reply:
Yup, then we agree